Dockerfile

```diff
@@ -45,8 +45,8 @@ RUN . ../asylum-venv/bin/activate && pip install -r requirements/local.txt
 
 # Configure application
 USER root
-COPY project /opt/asylum/
 RUN echo "DATABASE_URL=postgres://asylum:asylum@localhost/asylum" > .env
```

Comment:
Shouldn't this be `echo "DATABASE_URL=postgres://asylum:asylum@localhost/asylum" >> .env` so if there is an ENV file it gets appended instead of overwritten ??